src/TLuaInterpreter.cpp

```diff
@@ -19196,3 +19200,131 @@ void TLuaInterpreter::createCookiesTable(lua_State* L, QNetworkReply* reply)
     // Put "cookies" table into table (now 3 deep in stack), pop stack twice
     lua_settable(L, -3);
 }
+
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#getMapBackgroundColor
+int TLuaInterpreter::getMapBackgroundColor(lua_State* L)
+{
+    auto& host = getHostFromLua(L);
```

Comment:
This can just return the r,g,b colours. Lua has multiple return, so we don't have to create a table needlessly. See for example https://wiki.mudlet.org/w/Manual:Lua_Functions#getBgColor

Reply:
:thinking: Well, we return colours as triple-item list type tables as well don't we, I've checked and found:
* 2 in `getTextFormat(...)`
* 1 for each line in `getCustomLines(...)`
* 1 for each colour in `getCustomEnvColor()`

How many functions do a triple return for colours?

Reply:
That's because those have to be combined with another piece of data.

There is no reason to use a table here

Reply:
Okay, done.